clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/security/AzureNamedAccountCredentials.groovy

```diff
@@ -91,7 +94,7 @@ public class AzureNamedAccountCredentials implements AccountCredentials<AzureCre
 
     public AzureRegion(String name) {
       if (name == null) {
-        throw new NullPointerException("name");
+        throw new NullPointerException("name")
```

Comment:
Seems like a misuse of `NPE`, since I believe it's supposed to be thrown on access of a `null` object, not when one is discovered. Maybe an `IllegalArgumentException` is better?